src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java

```diff
@@ -120,15 +120,15 @@ public String getVoice() {
         return voice;
     }
 
-    public int getVoiceExtension() {
+    public String getVoiceExtension() {
```

Comment:
Could you please add Javadoc for this method? In the `@return` could you please mention that, a null value can be returned when not set.